oic.r.sensor/oic.r.sensor.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "http://openinterconnect.org/iotdatamodels/schemas/oic.r.sensor.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2016 Open Connectivity Foundation, Inc. All rights reserved.",
+  "title": "Generic Sensor",
+  "definitions": {
+    "oic.r.sensor": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "boolean",
+          "description": "ReadOnly, true = sensed, false = not sensed."
```

Comment:
This seems to be the standard for how it's done on oneiota

---
In reply to: [79096186](https://github.com/openT2T/translators/pull/97#discussion_r79096186) [](ancestors = 79096186)